lib/utils/ast.js

```diff
@@ -28,9 +28,34 @@ function hasAnnotation(node, annotation) {
   );
 }
 
+function isDescribeIdentifier(node) {
```

Comment:
Could combine this and `isPageIdentifier` similar to what we did `isCalleeProperty`

Reply:
Sure, updated.